pom.xml

```diff
@@ -9,7 +9,7 @@
 
 	<groupId>com.github.docker-java</groupId>
 	<artifactId>docker-java</artifactId>
-	<packaging>jar</packaging>
+	<packaging>bundle</packaging>
```

Comment:
why?

Reply:
From the plugin [documentation](http://felix.apache.org/documentation/subprojects/apache-felix-maven-bundle-plugin-bnd.html#using-the-plugin):
> There are two main things to note: (1) the <packaging> specifier must be "bundle" and (2) the plugin and configuration must be specified (the configuration section is where you will issue instructions to the plugin).

Reply:
frankly speaking i don't want changing packaging type of the project

Reply:
What if we create submodule?

Reply:
> frankly speaking i don't want changing packaging type of the project
why?

> What if we create submodule?
I don't know. I'm not a maven pro.